Import WPTs from folder html/canvas/element/text/

mbrodesser-Igalia opened the pull request:
#### 721e327ee60734808beba08b39a7de6934d5c948
<pre>
Import WPTs from folder html/canvas/element/text/
<a href="https://bugs.webkit.org/show_bug.cgi?id=290525">https://bugs.webkit.org/show_bug.cgi?id=290525</a>

Reviewed by Anne van Kesteren.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/035d520f54af6f9445287c64e8c891ac74a10c1b">https://github.com/web-platform-tests/wpt/commit/035d520f54af6f9445287c64e8c891ac74a10c1b</a>

Combined changes:
* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.align.default.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.align.invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.align.valid.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.baseline.default.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.baseline.invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.baseline.valid.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.direction.default-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.direction.default.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.baseline.default.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.direction.invalid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.direction.invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.direction.valid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.direction.valid.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.basic.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.align.center.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.align.end.ltr.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.align.end.rtl.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.align.left.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.align.right.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.align.start.ltr.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.align.start.rtl.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.baseline.alphabetic.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.baseline.bottom.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.baseline.hanging.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.baseline.ideographic.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.baseline.middle.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.baseline.top.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fill.maxWidth.NaN.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fill.maxWidth.bound.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fill.maxWidth.fontface.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fill.maxWidth.negative.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fill.maxWidth.small.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fill.maxWidth.zero.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fill.unaffected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fontface.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fontface.notinpage.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fontface.repeat.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.space.basic.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.space.collapse.end.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.space.collapse.nonspace.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.space.collapse.other.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.space.collapse.space.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.space.collapse.start.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.stroke.unaffected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.absolute.spacing.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.font-relative.spacing.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.fontKerning.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.fontKerning.with.uppercase.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.fontStretch.settings.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.fontVariant.settings.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.invalid.spacing.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.letterSpacing.change.font.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.letterSpacing.measure.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.measure.direction.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.measure.rtl.text.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.measure.textAlign.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.nonfinite.spacing.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.reset.TextRendering-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.reset.TextRendering.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.basic.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.reset.fontKerning.none-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.reset.fontKerning.none.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.fontKerning.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.reset.fontKerning.none2-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps5-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.reset.fontKerning.none2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.textRendering.settings.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.wordSpacing.change.font.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.wordSpacing.measure.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.default.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.basic.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.complex.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.complex2.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.family.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.size.percentage.default.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.size.percentage.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.system.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.tiny.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.relative_size.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.weight.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps.after.reset.font-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps.after.reset.font.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps1-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps1.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps2.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps3-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps3.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps4-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps4.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps5-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps5.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps6-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps6.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.lang.default-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.lang.default.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.align.default.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.lang.valid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.lang.valid.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.basic.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.actualBoundingBox.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.actualBoundingBox.whitespace-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.actualBoundingBox.whitespace.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.advances-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.advances.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.baselines.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.emHeights-low-ascent.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.emHeights-zero-descent.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.emHeights.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-align.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-align.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-baseline.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-baseline.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-drawing-styles-change.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-drawing-styles-change.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-font-change.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-font-change.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-options.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-options.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-range.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-range.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fontBoundingBox-reduced-ascent.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fontBoundingBox-zero-descent.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fontBoundingBox.ahem.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fontBoundingBox.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.getActualBoundingBox-exceptions.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.getActualBoundingBox-exceptions.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.getActualBoundingBox-full-text.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.getActualBoundingBox-full-text.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.getActualBoundingBox.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.getActualBoundingBox.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.index-from-offset-edge-cases.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.index-from-offset-edge-cases.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.index-from-offset-edges.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.index-from-offset-edges.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.index-from-offset.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.index-from-offset.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.lang-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.lang.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.width.empty.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.lang.inherit-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.lang.inherit.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.selection-rects-baselines.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.selection-rects-baselines.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.selection-rects-exceptions.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.selection-rects-exceptions.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.selection-rects.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.selection-rects.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-align.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-align.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-baseline.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-baseline.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-drawing-styles-change.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-drawing-styles-change.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-font-change.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-font-change.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-options.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-options.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-range.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-range.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.text-clusters-exceptions.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.text-clusters-exceptions.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.text-clusters-position.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.text-clusters-position.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.text-clusters-split.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.text-clusters-split.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.width.basic.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.width.empty.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.width.space.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.setFont.mathFont-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.setFont.mathFont.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.default.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.writingmode-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps5-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.writingmode.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/reference/direction-default-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/reference/direction-rtl-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/293002@main">https://commits.webkit.org/293002@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/bce9c48fe8c12b6e5824431556dd1e6916e275e7

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/97488 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/17113 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/7329 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/102575 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/48017 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/17406 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/25586 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/74285 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/31467 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/100491 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/13179 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/88161 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/54630 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/12948 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/6043 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/47459 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/82978 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/6124 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/104595 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/24567 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/17939 "Found 2 new test failures: imported/w3c/web-platform-tests/workers/data-url-shared.html ipc/send-gpu-GetShareableBitmap-RemoteRenderingBackend.html (failure)") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/83329 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/24939 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/84287 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/82750 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/20871 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/27293 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/4981 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/18157 "The change is no longer eligible for processing.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/24529 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/24351 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/27665 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/25925 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

annevk reviewed (approved):
r+ assuming this was not meant to be a draft and no tests expect to time out (those are skipped instead). Please confirm through a comment before merging.

Commit:
Import WPTs from folder html/canvas/element/text/
https://bugs.webkit.org/show_bug.cgi?id=290525

Reviewed by Anne van Kesteren.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/035d520f54af6f9445287c64e8c891ac74a10c1b

Combined changes:
* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.align.default.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.align.invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.align.valid.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.baseline.default.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.baseline.invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.baseline.valid.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.direction.default-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.direction.default.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.baseline.default.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.direction.invalid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.direction.invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.direction.valid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.direction.valid.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.basic.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.align.center.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.align.end.ltr.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.align.end.rtl.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.align.left.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.align.right.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.align.start.ltr.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.align.start.rtl.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.baseline.alphabetic.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.baseline.bottom.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.baseline.hanging.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.baseline.ideographic.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.baseline.middle.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.baseline.top.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fill.maxWidth.NaN.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fill.maxWidth.bound.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fill.maxWidth.fontface.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fill.maxWidth.negative.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fill.maxWidth.small.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fill.maxWidth.zero.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fill.unaffected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fontface.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fontface.notinpage.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.fontface.repeat.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.space.basic.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.space.collapse.end.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.space.collapse.nonspace.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.space.collapse.other.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.space.collapse.space.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.space.collapse.start.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.draw.stroke.unaffected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.absolute.spacing.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.font-relative.spacing.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.fontKerning.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.fontKerning.with.uppercase.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.fontStretch.settings.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.fontVariant.settings.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.invalid.spacing.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.letterSpacing.change.font.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.letterSpacing.measure.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.measure.direction.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.measure.rtl.text.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.measure.textAlign.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.nonfinite.spacing.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.reset.TextRendering-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.reset.TextRendering.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.basic.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.reset.fontKerning.none-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.reset.fontKerning.none.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.fontKerning.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.reset.fontKerning.none2-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps5-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.reset.fontKerning.none2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.textRendering.settings.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.wordSpacing.change.font.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.drawing.style.wordSpacing.measure.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.default.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.basic.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.complex.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.complex2.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.family.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.size.percentage.default.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.size.percentage.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.system.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.tiny.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.relative_size.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.weight.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps.after.reset.font-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps.after.reset.font.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps1-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps1.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps2.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps3-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps3.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps4-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps4.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps5-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps5.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps6-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps6.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.lang.default-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.lang.default.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.align.default.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.lang.valid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.lang.valid.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.parse.basic.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.actualBoundingBox.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.actualBoundingBox.whitespace-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.actualBoundingBox.whitespace.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.advances-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.advances.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.baselines.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.emHeights-low-ascent.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.emHeights-zero-descent.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.emHeights.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-align.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-align.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-baseline.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-baseline.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-drawing-styles-change.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-drawing-styles-change.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-font-change.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-font-change.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-options.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-options.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-range.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fillTextCluster-range.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fontBoundingBox-reduced-ascent.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fontBoundingBox-zero-descent.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fontBoundingBox.ahem.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.fontBoundingBox.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.getActualBoundingBox-exceptions.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.getActualBoundingBox-exceptions.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.getActualBoundingBox-full-text.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.getActualBoundingBox-full-text.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.getActualBoundingBox.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.getActualBoundingBox.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.index-from-offset-edge-cases.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.index-from-offset-edge-cases.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.index-from-offset-edges.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.index-from-offset-edges.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.index-from-offset.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.index-from-offset.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.lang-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.lang.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.width.empty.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.lang.inherit-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.lang.inherit.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.selection-rects-baselines.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.selection-rects-baselines.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.selection-rects-exceptions.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.selection-rects-exceptions.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.selection-rects.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.selection-rects.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-align.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-align.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-baseline.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-baseline.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-drawing-styles-change.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-drawing-styles-change.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-font-change.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-font-change.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-options.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-options.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-range.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.strokeTextCluster-range.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.text-clusters-exceptions.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.text-clusters-exceptions.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.text-clusters-position.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.text-clusters-position.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.text-clusters-split.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.text-clusters-split.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.width.basic.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.width.empty.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.width.space.html:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.setFont.mathFont-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.setFont.mathFont.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.font.default.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.writingmode-expected.html: Copied from LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.fontVariantCaps5-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.writingmode.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/reference/direction-default-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/reference/direction-rtl-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/w3c-import.log:

Canonical link: https://commits.webkit.org/293002@main
